src/api/tests/feature_tests/steps/endpoint_lambda_mapping.py

```diff
@@ -22,67 +22,25 @@ def get_endpoint_lambda_mapping() -> ENDPOINT_LAMBDA_MAPPING:
-            "ProductTeamEpr/{product_team_id}/ProductEpr": api.createEprProduct.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/DeviceReferenceData": api.createDeviceReferenceData.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/DeviceReferenceData/AccreditedSystemsAdditionalInteractions": api.createDeviceReferenceDataASActions.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/DeviceReferenceData/MhsMessageSet": api.createDeviceReferenceDataMessageSet.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/Device": api.createDevice.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/Device/MessageHandlingSystem": api.createDeviceMessageHandlingSystem.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/Device/AccreditedSystem": api.createDeviceAccreditedSystem.index,
         },
         "GET": {
             "ProductTeam/{product_team_id}": api.readProductTeam.index,
             "ProductTeam/{product_team_id}/Product/{product_id}": api.readCpmProduct.index,
-            "ProductTeamEpr/{product_team_id}": api.readProductTeamEpr.index,
-            # "ProductTeamEpr/{product_team_id}/Product": api.searchCpmProduct.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr": api.searchEprProduct.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}": api.readEprProduct.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/DeviceReferenceData": api.searchDeviceReferenceData.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/DeviceReferenceData/{device_reference_data_id}": api.readDeviceReferenceData.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}/{environment}/Device/{device_id}": api.readDevice.index,
-            "Questionnaire/{questionnaire_id}": api.readQuestionnaire.index,
-            "searchSdsDevice?nhs_id_code={nhs_id_code}": api.searchSdsDevice.index,
-            "searchSdsDevice?nhs_as_svc_ia={nhs_as_svc_ia}": api.searchSdsDevice.index,
-            "searchSdsDevice?nhs_id_code={nhs_id_code}&nhs_as_svc_ia={nhs_as_svc_ia}": api.searchSdsDevice.index,
-            "searchSdsDevice?nhs_id_code={nhs_id_code}&nhs_as_svc_ia={nhs_as_svc_ia}&foo={foo}": api.searchSdsDevice.index,
-            "searchSdsEndpoint": api.searchSdsEndpoint.index,
             "_status": api.status.index,
         },
         "DELETE": {
             "ProductTeam/{product_team_id}": api.deleteProductTeam.index,
-            "ProductTeamEpr/{product_team_id}/ProductEpr/{product_id}": api.deleteEprProduct.index,
             "ProductTeam/{product_team_id}/Product/{product_id}": api.deleteCpmProduct.index,
         },
```

Comment:
Depending on if this goes in before search we should remember to have Megan include the search one in here too :)